File: Pemrograman1/Modul5/Python/PRAK502-2310817310009-MuhammadFauzanAhsani.py
def mutlak(angka:int):
    return abs(angka)

def hitung(nilai1:int, nilai2:int):
    return mutlak(nilai1 - nilai2)

a, c, b, d = map(int, input().split())
Hasil = hitung(a,b) + hitung(c,d)
print(mutlak(Hasil))

File: Pemrograman1/Modul5/C/PRAK502-2310817310009-MuhammadFauzanAhsani.c
#include <stdio.h>
#include <math.h>

int mutlak(int angka){
    return abs(angka);
}

int hitung(int nilai1, int nilai2){
    return mutlak(nilai1 - nilai2);
}

int main(){
    int a, b, c, d;
    scanf("%d %d %d %d", &a, &c, &b, &d);

    int Hasil = hitung(a,b) + hitung(c, d);
    printf("%d", mutlak(Hasil)); 
    return 0;
}